RUFAS/output_manager.py

```diff
@@ -549,25 +549,86 @@ def _dict_to_csv_column_list(self, variable_name: str, data_dict: Dict[str, List
             A list of (column_name, column_data) tuples.
 
         """
+
         column_list = []
-        mandatory_fields = ["values", "info_maps"] if "info_maps" in data_dict else ["values"]
-        for field in mandatory_fields:
-            data_list = data_dict[field]
-            if data_list and isinstance(data_list[0], dict):
-                csv_column_lists: Dict[str, List[Any]] = {subkey: [] for item in data_list for subkey in item.keys()}
-                for nested_dictionary in data_list:
-                    for subkey, value in nested_dictionary.items():
-                        csv_column_lists[subkey].append(value)
-
-                for subkey in csv_column_lists.keys():
-                    column_title = f"{variable_name}.{subkey}"
-                    column_list.append(pd.Series(csv_column_lists[subkey], dtype=object, name=column_title))
-            else:
-                column_title = f"{variable_name}"
-                column_list.append(pd.Series(data_list, dtype=object, name=column_title))
+        units = data_dict["info_maps"][0]["units"] if data_dict.get("info_maps", []) else None
+        data_list = data_dict["values"]
+        if data_list and isinstance(data_list[0], dict):
+            csv_column_lists: Dict[str, List[Any]] = {subkey: [] for item in data_list for subkey in item.keys()}
```

Comment:
```suggestion
            csv_column_lists: Dict[str, List[Any]] = {
            subkey: [] 
            for item in data_list 
            for subkey in item.keys()
}
```
More readable if this is written in multiple lines.

Reply:
Ugh, this unfortunately isn't my code. It looks like I may have accidentally rebased incorrectly and pulled in changes and commits from another PR.